### PR TITLE
fix:NumberTheory/LegendreSymbol/GaussSum fix porting note

### PR DESCRIPTION
**Pull request by xroblot**
~~This PR incorporates also the changes to `NumberTheory.LegendreSymbol.MulCharacter` discussed [here](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/coercion.20issue.20in.20NumberTheory.2ELegendreSymbol.2EMulCharacter/near/343943344)~~

This PR has been reopened because I forgot to write a porting note 

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/NumberTheory/LegendreSymbol/GaussSum.lean [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+    gaussSum χ ψ ^ p = gaussSum (χ ^ p) (ψ ^ p) := by
+  rw [← frobenius_def, gaussSum, gaussSum, map_sum]
+  simp_rw [pow_apply' χ fp.1.pos, map_mul, frobenius_def]
+  rfl
+#align gauss_sum_frob gaussSum_frob
+
+/-- For a quadratic character `χ` and when the characteristic `p` of the target ring
+is a unit in the source ring, the `p`th power of the Gauss sum of`χ` and `ψ` is
+`χ p` times the original Gauss sum. -/
+-- Porting note: TODO
```
Comment (Ruben-VandeVelde) [inline Mathlib/NumberTheory/LegendreSymbol/GaussSum.lean:166]: What is todo?

Comment (xroblot) [reply Mathlib/NumberTheory/LegendreSymbol/GaussSum.lean:166]: Oh yes, I forgot to say that I added `norec`

Comment (xroblot) [reply Mathlib/NumberTheory/LegendreSymbol/GaussSum.lean:166]: I guess there is no way to modify this PR once it's merged...

**Comment by ChrisHughes24** [comment]
bors merge

**Pull request closed without merging**